recipe/meta.yaml

```diff
@@ -41,7 +41,7 @@ requirements:
     - setuptools
     - llvmlite 0.40.*
     - numpy
-    - tbb-devel 2021.6.0
+    - tbb-devel >=2021.6.0
```

Comment:
Could you please provide some more context for this change?

The pin is to ensure that we build against this version of `tbb-devel`. As noted in `run_constrained` users then get a version of `tbb` that has this minimum and floats upwards

Sorry, but think we should revert this

Reply:
run_constrained is also changed in the recipe has a slightly different https://github.com/numba/numba/blob/0.57.0/buildscripts/condarecipe.local/meta.yaml#L51. The problem is that this change should give users the opportunity to use tbb of any version starting from 2021.6. So the best way here is to rely on numba recipe and not reverse this change

Reply:
Thanks for the additional context and digging into this further! 🙏

Generally this seems right. Though it would be nice to avoid hard-coding the version (to ensure we don't miss updating it in the future)

Looking at [the conda-build `pin_compatible` docs]( https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2-functions ). Maybe we can try?

```diff
   run_constrained: 
-    - {{ pin_compatible('tbb') }} 
+    - {{ pin_compatible('tbb', max_pin=None) }} 
```

Reply:
Submitted PR ( https://github.com/conda-forge/numba-feedstock/pull/124 ) with that change. Please take a look :)